readinglist/resource.py

```diff
@@ -93,6 +95,12 @@ class RessourceSchema(colander.MappingSchema):
     _id = colander.SchemaNode(colander.String(), missing=colander.drop)
     last_modified = TimeStamp()
 
+    __readonly_fields__ = ('_id', 'last_modified')
```

Comment:
I actually quite like the `__readonly_fields__`, I find its meaning obvious, and says to me "this is a metadata field". I don't know where this is coming from though :D